ts/components/session/settings/SessionSettings.tsx

```diff
@@ -0,0 +1,191 @@
+import React from 'react';
+
+import { SettingsHeader } from './SessionSettingsHeader';
+import { SessionSettingListItem } from './SessionSettingListItem';
+
+export enum SessionSettingCategory {
+  General = 'general',
+  Account = 'account',
+  Privacy = 'privacy',
+  Permissions = 'permissions',
+  Notifications = 'notifications',
+  Devices = 'devices',
+}
+
+export enum SessionSettingType {
+  Toggle = 'toggle',
+  Options = 'options',
+  Button = 'button',
+  Slider = 'slider',
+}
+
+//const { Settings } = window.Signal.Types;
+
+export interface SettingsViewProps {
+  category: SessionSettingCategory;
+}
+
+export class SettingsView extends React.Component<SettingsViewProps> {
+  public settingsViewRef: React.RefObject<HTMLDivElement>;
+
+  public constructor(props: any) {
+    super(props);
+    this.settingsViewRef = React.createRef();
+  }
+
+  public renderSettingInCategory() {
+    const { Settings } = window.Signal.Types;
+
+    // Grab initial values from database on startup
+    // ID corresponds to instalGetter parameters in preload.js
+    // They are NOT arbitrary; add with caution
+    const localSettings = [
+      {
+        id: 'theme-setting',
+        title: window.i18n('themeToggleTitle'),
+        description: 'Choose the theme best suited to you',
```

Comment:
You missed that one ! (I think I saw it on the json translation file, so you are just missing the `i18n` call)